src/server/frontend_wayland/window_wl_surface_role.h

```diff
@@ -157,6 +159,13 @@ class WindowWlSurfaceRole
 
     std::unique_ptr<shell::SurfaceSpecification> pending_changes;
 
+    struct TrackedOutput
+    {
+        mir::graphics::DisplayConfigurationOutput config;
+        bool bound{true};
```

Comment:
Somehow `bound` seems misleading as it is more an "in use", sometimes reflecting whether we have "entered" at some time in the past and sometimes is a working variable.